processor/batchprocessor/batch_processor.go

```diff
@@ -23,12 +23,14 @@ import (
 
 	"go.uber.org/zap"
 
+	"go.opentelemetry.io/collector/client"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/pdata/plog"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.opentelemetry.io/collector/pdata/ptrace"
 	"go.opentelemetry.io/collector/processor"
+	"go.opentelemetry.io/otel/attribute"
```

Comment:
I can, and I will, but I in passing I want to state that:

- why not use this (stable, high-performance, battle-tested API in the OTel-Go repo)?
- it's outrageous not to have a utility function available for use across the collector code base for exactly this purpose. ESPECIALLY since you've removed the ability to stable-sort a pcommon.Map in place, IMO.

Reply:
[fd9a671](https://github.com/open-telemetry/opentelemetry-collector/pull/7325/commits/fd9a671049de8ecbf12036546465c67269a02c6e)